controllers/cookies.js

```diff
@@ -0,0 +1,89 @@
+const models = require('../models')
+
+const cookiesPage = async (request, response) => {
+  const api = await models.cookies.findAll()
+
+  return response.render('index', { api })
+}
+
+const getAllCookies = async (request, response) => {
+  try {
+    const allCookies = await models.cookies.findAll()
+
+    return response.send(allCookies)
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve cookies, please try again')
+  }
+}
+
+const saveNewCookie = async (request, response) => {
+  try {
+    const {
+      name, description, aboutId
```

Comment:
You're expecting a user to supply an aboutId but the concept of an about is internal to your system, your API doesn't expose the concept to the user. There is no way for the user to know which id to provide and also no way for them to create a new about record. This route should work the way your docs originally defined it.